docs/cli/bundlestore.md

```diff
@@ -84,4 +84,4 @@ Disabling the bundle store will automatically restore settings that were in use
 [bundlestore use]: ./bundlestore/use.md
 [bundlestore delete]: ./bundlestore/delete.md
 [bundlestore upload]: ./bundlestore/upload.md
-[Electrode Native bundle store server]: https://github.com/electrode-io/ern-bundle-store
\ No newline at end of file
+[Electrode Native bundle store server]:https://github.com/electrode-io/ern-bundle-store
```

Comment:
No need to remove the space after `:` here. (probably unintentional?)

Reply:
Yes